bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewRepeaterHandler.java

```diff
@@ -0,0 +1,218 @@
+import org.openhab.binding.hdpowerview.internal.api.Firmware;
+import org.openhab.binding.hdpowerview.internal.api.responses.RepeaterData;
+import org.openhab.binding.hdpowerview.internal.config.HDPowerViewRepeaterConfiguration;
+import org.openhab.binding.hdpowerview.internal.exceptions.HubException;
+import org.openhab.binding.hdpowerview.internal.exceptions.HubInvalidResponseException;
+import org.openhab.binding.hdpowerview.internal.exceptions.HubMaintenanceException;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.UnDefType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handles commands for an HD PowerView Repeater
+ *
+ * @author Jacob Laursen - Initial contribution
+ */
+@NonNullByDefault
+public class HDPowerViewRepeaterHandler extends AbstractHubbedThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(HDPowerViewRepeaterHandler.class);
+
+    private static final int REFRESH_INTERVAL_MINUTES = 5;
+    private static final int IDENTITY_PERIOD_SECONDS = 3;
+    private static final String COMMAND_IDENTIFY = "IDENTIFY";
+
+    private @Nullable ScheduledFuture<?> refreshStatusFuture = null;
+    private @Nullable ScheduledFuture<?> resetIdentifyStateFuture = null;
+    private int repeaterId;
+
+    public HDPowerViewRepeaterHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        repeaterId = getConfigAs(HDPowerViewRepeaterConfiguration.class).id;
+        logger.debug("Initializing repeater handler for repeater {}", repeaterId);
+        if (repeaterId <= 0) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "@text/offline.conf-error.invalid-id");
+            return;
+        }
+        Bridge bridge = getBridge();
+        if (bridge == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED);
+            return;
+        }
+        if (!(bridge.getHandler() instanceof HDPowerViewHubHandler)) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED,
+                    "@text/offline.conf-error.invalid-bridge-handler");
+            return;
+        }
+        ThingStatus bridgeStatus = bridge.getStatus();
+        if (bridgeStatus == ThingStatus.ONLINE) {
+            updateStatus(ThingStatus.UNKNOWN);
+        } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+        }
+        scheduleRefreshJob();
+    }
+
+    @Override
+    public void dispose() {
+        logger.debug("Disposing repeater handler for repeater {}", repeaterId);
```

Comment:
This is already logged by the framework and can therefore be removed.

Reply:
I can not find this being logged? At least not anything including the ID.

Reply:
Do you indirectly like this?
```
2022-01-24 21:45:32.507 [DEBUG] [l.handler.HDPowerViewRepeaterHandler] - Disposing repeater handler for repeater 17805
2022-01-24 21:45:32.520 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'hdpowerview:repeater:hub:repeater' changed from UNINITIALIZED to UNINITIALIZED (HANDLER_MISSING_ERROR)
```

Maybe it would still be useful with the distinct dispose() logging including ID as long as it's only with debug level?

Reply:
OK